### ISSUE TITLE
[23][IDE Support] JEP 467: Markdown Documentation Comments #2429

### ISSUE THREAD

**Issue by mpalat** [eclipse-jdt/eclipse.jdt.core#2845]
IDE Support

DOM AST + Rewrite
Search
Selection
Content Assist
Formatter
Create issues as required.

**Comment by stephan-herrmann** [comment]
> IDE Support
> 
> * DOM AST + Rewrite

No new AST elements, so no apparent need for further changes.

> * Search
> * Selection

Since references are created like in traditional javadoc, it might "just work". Smoke test for search & selection in a runtime IDE looks good. Let's just create a few tests.

> * Content Assist

See
* https://github.com/eclipse-jdt/eclipse.jdt.core/issues/2744
  * I believe this is blocked by https://github.com/eclipse-jdt/eclipse.jdt.ui/issues/1542 , right @jarthana ?

> * Formatter

New issue #2851

**Comment by jarthana** [comment]
>       * I believe this is blocked by [[23] Partition support for markdown comments eclipse.jdt.ui#1542](https://github.com/eclipse-jdt/eclipse.jdt.ui/issues/1542) , right @jarthana ?

I had released a partial fix in jdt.core for completion of Java elements from jdt.core as part of @see etc. Now, with the https://github.com/eclipse-jdt/eclipse.jdt.ui/pull/1549, I have added some more code in jdt.ui that offers tag template proposals from jdt.ui. If time permits, please review that updated PR and let me know your findings. TIA!

**Comment by stephan-herrmann** [comment]
For a first milestone I declare this issue completed.
